workos/resources/directory_sync.py

```diff
@@ -75,6 +75,7 @@ class WorkOSDirectoryUser(WorkOSBaseResource):
         "organization_id",
         "first_name",
         "last_name",
+        "job_title",
```

Comment:
Does this already correctly account for nullable fields?

Reply:
Yes, it looks like this update sets the field to `None` in that case: https://github.com/workos/workos-python/pull/103